service/docs/source/geopmaccess.1.rst

```diff
@@ -1,28 +1,181 @@
-.. role:: raw-html-m2r(raw)
-   :format: html
 
-
-geopmaccess(1) -- Access management for the geopm service
+geopmaccess(1) -- Access management for the GEOPM Service
 =========================================================
 
+SYNOPSIS
+--------
+
+
+Read Access List
+~~~~~~~~~~~~~~~~
+
+.. code-block:: none
+
+    geopmaccess [-c] [-g GROUP | -a]
+
+
+Write Access List
+~~~~~~~~~~~~~~~~~
+
+.. code-block:: none
+
+    geopmaccess -w [-c] [-g GROUP] [-n | -s]
+
 
+Remove Access List
+~~~~~~~~~~~~~~~~~~
 
+.. code-block:: none
 
+    geopmaccess -D [-c] [-g GROUP] [-n]
+
+
+
+Get Help or Version
+~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: none
+
+    geopmaccess -h | -v
 
 
 DESCRIPTION
 -----------
 
-Access management for the geopm service.
+The GEOPM service uses the ``/etc/geopm-service`` directory to store
+files that control which signals and controls may be accessed by a
+user through the service.  The purpose of the ``geopmaccess`` command
+line tool is to enable reading and writing of these access list files.
+
+
+Options
+~~~~~~~
+
+*
+  ``-c``, ``--controls``:
+  Command applies to controls not signals.
+
+
+* ``-u``, ``--user``:
+  Read the default user access list.
+
+*
+  ``-g``, ``--group``:
+  Read or write the access list for a specific Unix GROUP.
+
+*
+  ``-a``, ``--all``:
+  Print all signals or controls supported by the service system (not
+  valid with ``-w`` option).
+
+*
+  ``-w``, ``--write``:
+  Write default access list, or the access list for a particular Unix
+  group.
+
+*
+  ``-D``, ``--delete``:
+  Remove an access list for default user or a particular Unix Group.
+*
+  ``-n``, ``--dry-run``:
+  Do error checking on all user input, but do not modify configuration
+  files
+
+*
+  ``-E``, ``--skip-check``:
```

Comment:
Maybe `-q` `--quiet`?

Reply:
Nevermind. Reached the description at the end of the man page, and I see this is different from a quiet mode. Maybe `-f`, `--force` fits other CLIs that allow similar validation skips.